include/avif/avif.h

```diff
@@ -526,7 +526,10 @@ typedef struct avifReformatState
     int rgbMaxChannel;
     float yuvMaxChannelF;
     float rgbMaxChannelF;
-    int uvBias; // the integer value of 0.5 for the appropriate bit depth [128, 512, 2048]
+    float yBias; // minimum Y value
+    float uvBias; // the value of 0.5 for the appropriate bit depth [128, 512, 2048]
+    float yRange; // difference between max and min Y
+    float uvRange; // difference between max and min UV
```

Comment:
Nit/optional: yRange and uvRange look like the yuvRange member at line 524 above, but they have different meanings. This could be confusing, but it may not be a real problem. We can address this by  changing yRange and uvRange to rangeY and rangeUV. If we do that, it would be good to also change yBias and uvBias to biasY and biasUV for consistency.